buildchain/buildchain/salt_tree.py

```diff
@@ -425,7 +425,10 @@ def _get_parts(self) -> Iterator[str]:
     Path("salt/metalk8s/addons/nginx-ingress-control-plane/certs/init.sls"),
     Path("salt/metalk8s/addons/nginx-ingress-control-plane/certs/server.sls"),
     Path("salt/metalk8s/addons/nginx-ingress-control-plane/deployed/init.sls"),
-    Path("salt/metalk8s/addons/nginx-ingress-control-plane/deployed/chart.sls"),
+    Path(
+        "salt/metalk8s/addons/nginx-ingress-control-plane/deployed/",
+        "chart-daemonset.sls",
+    ),
```

Comment:
Today they are not really different (mainly Service ClusterIP vs LoadBalancer and Deployment vs DaemonSet) but at some point, it may have some logic in the helm chart that makes the resulting "chart" really different.

I agree that we could have something smarter here to do not duplicate things in different SLS files but to me, with the current status of helm chart management, we have in MetalK8s this approach is the simpler/better(? maybe) one